src/vm/docs/mod.rs

```diff
@@ -1831,6 +1833,13 @@ mod test {
                 .unwrap(),
             )
         }
+        fn get_stacks_epoch(&self, height: u32) -> Option<StacksEpoch> {
+            Some(StacksEpoch {
+                epoch_id: StacksEpochId::Epoch20,
+                start_height: 0,
+                end_height: u64::MAX,
```

Comment:
Should this be `i64::MAX as u64`?

Reply:
Fixed (but it doesn't matter in this context because it'll never hit the database)